app/models/concerns/bank_account_validations.rb

```
@@ -0,0 +1,112 @@
+
+      if iban_countries.include?(self.class.name)
+        return if Ibandit::IBAN.new(account_number_decrypted).valid?
+        errors.add :base, "The account number is invalid."
+        return
+      end
+
+      return unless self.class.const_defined?(:ACCOUNT_NUMBER_FORMAT_REGEX, false)
+      return if self.class.const_get(:ACCOUNT_NUMBER_FORMAT_REGEX).match?(account_number_decrypted.to_s)
+
+      errors.add :base, "The account number is invalid."
+    end
```

RuboCop (1.75.5)

[convention] Inconsistent indentation detected.

(Layout/IndentationConsistency)